ee/modules/502-delivery/openapi/config-values.yaml

```
@@ -0,0 +1,113 @@
+type: object
```

```suggestion
x-config-version: 1
type: object
```
It is 1 by default, but why not to add it explicilty for the new config-values.yaml (there should be linter for this, yeah)